### PR TITLE
Update for CLAP 1.1.8

### PR DESCRIPTION
**Pull request by robbert-vdh**
Based on https://github.com/free-audio/clap/compare/1.1.7...1.1.8. Includes #25.

### CODE REVIEW

File: src/ext/draft/configurable_audio_ports.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct clap_audio_port_configuration_request {
+    pub is_best_effort: bool,
+    pub is_input: bool,
+    pub port_index: u32,
+    pub channel_count: u32,
+    pub port_type: *const c_char,
+    pub port_details: *const c_char,
```
Comment (micahrj) [inline src/ext/draft/configurable_audio_ports.rs]: Should be `*const c_void`: https://github.com/free-audio/clap/blob/41964fa3f44fdbf6feb20df528a9cdf2cc3b05ba/include/clap/ext/draft/configurable-audio-ports.h#L34

File: src/ext/draft/extensible_audio_ports.rs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            channel_count: u32,
+            port_type: *const c_char,
+            port_details: *const c_void,
+        ) -> bool,
+    >,
+    pub remove_port: Option<
+        unsafe extern "C" fn(
+            plugin: *const clap_plugin,
+            is_input: bool,
+            channel_count: u32,
```
Comment (micahrj) [inline src/ext/draft/extensible_audio_ports.rs]: Should be `index: u32`: https://github.com/free-audio/clap/blob/41964fa3f44fdbf6feb20df528a9cdf2cc3b05ba/include/clap/ext/draft/extensible-audio-ports.h#L28

**Review by micahrj — CHANGES_REQUESTED** [review]
Apologies for taking so long to review this.

File: src/ext/draft/configurable_audio_ports.rs (hunk 1 above)
Comment (robbert-vdh) [reply src/ext/draft/configurable_audio_ports.rs]: Good catch, thanks! Fixed this and the other one.

**Review by micahrj — APPROVED** [review]

**Pull request merged**